extensions/hazelcast-client/runtime/src/main/java/io/quarkus/hazelcast/client/runtime/HazelcastClientConfig.java

```diff
@@ -0,0 +1,65 @@
+package io.quarkus.hazelcast.client.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "hazelcast-client", phase = ConfigPhase.RUN_TIME)
+public class HazelcastClientConfig {
+
+    /**
+     * Hazelcast Cluster address
+     */
+    @ConfigItem
+    public Optional<String> clusterMembers;
```

Comment:
Can't they be a list of `InetSocketAddress`?

It would be better to not have it optional and have a default value instead (if possible). That way, it's automatically documented.